actions/install/installer.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# SLSA verifier install script
+
+set -euo pipefail
+shopt -s expand_aliases
+alias log_info="echo \"INFO:\""
+alias log_error="echo \"ERROR:\""
+
+verifier_release="$1"
+install_dir="$HOME/.slsa/bin/${verifier_release}"
+
+mkdir -p "$install_dir"
+cd "$install_dir"
+
+if [[ "$verifier_release" == "main" ]]; then
```

Comment:
@ianlewis is it standard to support "main"?

Reply:
I'm not sure about all `setup-*` actions. I think the general convention is to support a "latest" special case and install the "latest" release but I don't think "main" is usually supported. From what I can tell [setup-go](https://github.com/actions/setup-go) only supports versions in a specific [manifest file](https://github.com/actions/go-versions/blob/main/versions-manifest.json) or you can omit the input and it installs the latest release. [setup-node](https://github.com/actions/setup-node) seems to support `latest`/`current`/`node` as a keyword to install the latest release. But neither support installing from `main` as far as I can tell.

In our case, I think we should not support it or, at the very least, strongly discourage it.

Reply:
I think we should disallow support: we often may change API/change semantics at main and need to notify users at a next major release if there's breaking changes. allowing main would mean that users would pick up on breaking changes without notification